src/vs/editor/contrib/linkedEditing/browser/linkedEditing.css

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
As this is a new css file, don't forget to add the css import in src/vs/editor/contrib/linkedEditing/browser/linkedEditing.ts

`import 'vs/css!./linkedEditing';